Consul.Test/KVTest.cs

```diff
@@ -37,8 +37,12 @@ internal static string GenerateTestKeyName()
 
             for (var i = 0; i < keyChars.Length; i++)
             {
-                keyChars[i] = Convert.ToChar(Random.Next(65, 91));
+                lock (Random)
```

Comment:
`Random` class is not thread-safe, we need to make sure only one thread uses it at a time.